packages/rest/test/unit/rest-server/rest-server.test.ts

```diff
@@ -71,6 +71,14 @@ describe('RestServer', () => {
       expect(server.getSync(RestBindings.PORT)).to.equal(3000);
     });
 
+    it('uses http host localhost by default', async () => {
+      const app = new Application({
+        components: [RestComponent],
+      });
+      const server = await app.getServer(RestServer);
+      expect(server.getSync(RestBindings.HOST)).to.equal('localhost');
```

Comment:
If the test is already using an async handler, you can use `server.get` here instead.
`expect(await server.get(RestBindings.HOST)).to.equal('localhost');`

Reply:
Good catch. A copy/paste problem.